src/TLuaInterpreter.cpp

```diff
@@ -10357,24 +10357,20 @@ int TLuaInterpreter::registerAnonymousEventHandler(lua_State* L)
 }
 
 
-int TLuaInterpreter::Send(lua_State* L)
+int TLuaInterpreter::expandAlias(lua_State *L)
 {
     string luaSendText;
     if (!lua_isstring(L, 1)) {
-        lua_pushstring(L, "Send: wrong argument type");
+        lua_pushstring(L, "expandAlias: wrong argument type");
         lua_error(L);
         return 1;
     } else {
         luaSendText = lua_tostring(L, 1);
     }
     bool wantPrint = true;
     if (lua_gettop(L) > 1) {
-        if (!lua_isboolean(L, 2)) {
-            lua_pushstring(L, "Send: wrong argument type");
-            lua_error(L);
-            return 1;
-        } else {
-            wantPrint = lua_toboolean(L, 2);
+        if (lua_isboolean(L, 2) && !lua_toboolean(L, 2)) {
```

Comment:
Checking that the argument can be read as a boolean and that it is `false` - isn't it? How does this compare to that other PR where some arguments are numbers but still used as a boolean - which is _best_?

Reply:
This is making sure it's `false` and not a `nil`. Using numbers as a boolean is suboptimal, but that PR does not introduce that - that is current behaviour of the function.

Reply:
I've added a comment to help explain. This is a Lua API specific though.